oop/music/test_indie_synth.py

```diff
@@ -0,0 +1,30 @@
+from oop.music.music import Music
+from oop.music.lyrics import Lyrics
+from oop.music.indie import Indie
+from oop.music.synth import Synth
+
+def test_indie():
+    """ verifying an instance of indie music """
+    autoheart = Indie('Juggernaut', 124, ['vocals'], 'Like a juggernaut, see it coming', True)
+    assert autoheart.play() == 'Juggernaut is a song that plays at 124 beats per minute with vocals'
+    assert autoheart.listen() == 'The song Juggernaut has the lyrics Like a juggernaut, see it coming'
+    assert autoheart.words() == 'it is True that Juggernaut has lyrics'
+
+    assert autoheart.bpm == 124
+    assert autoheart.has_lyrics == True
+    assert isinstance (autoheart, Music)
```

Comment:
correctly demonstrates polymorphism 👍 watch style -- should be assert isinstance(autoheart, Music) no spaces